controllers/builder/containerbuilder.go

```diff
@@ -200,6 +206,12 @@ func newBuild(buildInput kanikoBuildInput, platform api.PlatformContainerBuild,
 		newBuilder.AddConfigMapResource(res.ConfigMap, res.WorkflowPath)
 	}
 
+	//TODO WM, remove this comment.
+	//make the workflow properties available to the kaniko build.
+	for _, props := range buildInput.workflowProperties {
+		newBuilder.AddConfigMapResource(props.ConfigMap, props.WorkflowPath)
```

Comment:
@ricardozanini So, the idea is:
At the time of launching the kaniko build, we configure additionally the application.properties and the managed properties. In that way, both properties are available at build time.